src/features/auth/ui/GoogleLoginButton.module.css

```
@@ -0,0 +1,108 @@
+.gsi-material-button {
+    position: relative;
+    z-index: 2;
+    -moz-user-select: none;
+    -webkit-user-select: none;
+    -ms-user-select: none;
+    -webkit-appearance: none;
+    background-color: WHITE;
+    background-image: none;
+    /*border: 1px solid #747775;*/
```

Stylelint (17.9.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)